content/building-blocks/analyze-data/non-parametric-tests/introduction.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Intro to non parametric tests"
+description: "Describing the types of non parametric tests and their usage"
+keywords: "non, para, test, binom, chi, wilcoxon, mcnemar, fisher, median, mann, whitney, kruskal, wallis"
+weight: 1
+date:
+draft: false
+aliases:
+  - /non-para-intro
+---
+
+## Overview
+
+Performing statistical tests like t-test and f-test are imperative in any empirical study to help test the hypotheses about population parameters. However, parametric tests like these make certain assumptions about the underlying population distribution which are not always reasonable to make. Hence, in such a case, one should use a non-parametric test instead.
+In this building block, we will describe the various non-parametric test that you can use according to the data sample you have.
+
+## Which test to use?
```

Comment:
Can you move this part in a tip box, similar to how this is done here?

https://tilburgsciencehub.com/building-blocks/automate-and-execute-your-work/reproducible-work/config-vm-gcp/

This will help increase/improve the flow of this section.